### PR TITLE
[#247] Footer arrangement on mobile

### LINKED ISSUE #247: Footer Arrangement on Mobile
Currently the footer social icons will collapse onto the next line if the screen size is the same. Instead lets have them shrink in size instead so they still fit on the same line

### PR DESCRIPTION
**Pull request by NovaSagittarii**
New footer behavior, use `text-5vw` instead of `text-5xl`

image
image
image
image

this should be fine since i don't think this is too big, maybe can use min() to cap size? (tailwind doesnt really support min though so this might have to be done through setting a style)

### CODE REVIEW

File: components/Footer.js [hunk 1]
```diff
@@ -30,7 +29,11 @@ const Footer = () => {
 										title={link.title}
 										className='fill-current text-acm-white hover:text-acm-blue'
 									>
-										{link.image}
+										{(function () {
```
Comment (shahdivyank) [inline components/Footer.js]: 💀 what is this 💀

File: components/data/Socials.js [hunk 2]
```diff
@@ -7,36 +7,37 @@ import {
 	FaGithub,
 } from "react-icons/fa";
 
+const classNames = "text-[5vw]";
```
Comment (shahdivyank) [inline components/data/Socials.js]: put this directly into the className, it does get repetitive but cleaner to understand cause everything you need is within that line

File: components/Footer.js (hunk 1 above)
Comment (NovaSagittarii) [reply components/Footer.js]: OH LMAO  ITS DEBUG CODE XDDDDDDDDDDDDDDD
i was trying to see if i could modify a style after the component was called but before it was inserted (it didnt work)

File: components/data/Socials.js (hunk 2 above)
Comment (NovaSagittarii) [reply components/data/Socials.js]: oh okay

**Review by shahdivyank — APPROVED** [review]

**Pull request merged**